infra/src/uber-charts/islandis.ts

```diff
@@ -108,6 +110,7 @@ export const Services: EnvironmentServices = {
     appSystemApiWorker,
     userNotificationService,
     userNotificationWorkerService,
+    contentfulEntryTagger,
```

Comment:
You should probably start by having the service added only to the dev stack, while it is being developed

Reply:
Ok, yep I agree